src/utils/capitalizeFirstLetter.ts

```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (text: string | undefined) => {
+  return text !== undefined && text.charAt(0).toUpperCase() + text.slice(1);
```

Comment:
This looks fine as you want to do the capitalisation in the hook. Just be aware you can do this with CSS easily too and its often good to think about when its best to use either approach

```
&:first-letter{
  text-transform: capitalize
}
```

Reply:
Thats a good point. I agree that this approach is a bit better just for terms of simplicity.

Reply:
Great point. I followed you advice!